src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -137,7 +146,7 @@ const ResourceCollection = ({
 }) => {
   return (
     <>
-      {resourceList.map((resource: any, index: number) => {
+      {resourceList.map((resource: NestedResource, index: number) => {
```

Comment:
this could be as simple as `{type: string, resourceList: Resource[]}` since those are the properties you need below, then it'd be only restrictive in the sense that "whatever I pass in here needs to look like this" versus a larger more complete structure that won't even be accessed.